src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -616,7 +618,8 @@ protected virtual void MarkUserDependency (MemberReference context, CustomAttrib
 			if (MarkDependencyField (td, member, new DependencyInfo (DependencyKind.PreservedDependency, ca)))
 				return;
 
-			_context.LogMessage (MessageImportance.High, $"Could not resolve dependency member '{member}' declared in type '{td.FullName}'");
+			// TODO: Add origin, basically the place where `PreserveDependencyAttribute` is used.
```

Comment:
Either create an issue or resolve the TODO... ideally we should not have TODOs in the code - they are not really useful (And nobody tracks them).